platform/android/gradle/gradle-javadoc.gradle

```diff
@@ -2,17 +2,24 @@ android.libraryVariants.all { variant ->
     def name = variant.name
     task "javadoc$name"(type: Javadoc) {
         description = "Generates javadoc for build $name"
-        failOnError = false
         destinationDir = new File(destinationDir, variant.baseName)
         source = variant.sourceSets.collect { it.java.sourceFiles }.inject { m, i -> m + i }
+        classpath += project.files(android.getBootClasspath().join(File.pathSeparator))
         options.windowTitle("Mapbox Maps SDK for Android $VERSION_NAME Reference")
         options.docTitle("Mapbox Maps SDK for Android $VERSION_NAME")
         options.header("Mapbox Maps SDK for Android $VERSION_NAME Reference")
         options.bottom("&copy; 2015&ndash;2019 Mapbox. All rights reserved.")
         options.links("http://docs.oracle.com/javase/7/docs/api/")
-        options.linksOffline("http://d.android.com/reference/", "$System.env.ANDROID_HOME/docs/reference")
+        options.linksOffline "http://d.android.com/reference","${android.sdkDirectory}/docs/reference"
         options.overview("src/main/java/overview.html")
         options.group("Mapbox Android SDK", "com.mapbox.*")
-        exclude '**/R.java', '**/BuildConfig.java'
+        exclude '**/R.java', '**/BuildConfig.java', '**/**.java.ejs', '**/**.html'
     }
 }
+
+afterEvaluate {
```

Comment:
This code block makes the licenses script fail :thinking:

Reply:
![tenor](https://user-images.githubusercontent.com/2151639/58882324-48928600-86dc-11e9-9998-559985f45a05.gif)

Reply:
https://github.com/mapbox/mapbox-gl-native/pull/14792/files#diff-bd744698440e628074ff435021cb4945R7 works for both

Reply:
:man_dancing: